### ISSUE TITLE
User card has odd margin attribute

### ISSUE THREAD

**Issue by amotl** [Daemonite/discourse-material-theme#16]
## Problem
The user card currently has a large spacing due to some margin setting which seems to be nailed to `80px`.

![image](https://user-images.githubusercontent.com/453543/51088143-9a9e3380-175c-11e9-91c1-18295f9e1ba4.png)

**Comment by amotl** [comment]
## Workaround
This patch
```css
/* Fix odd margin attribute with user card (DMT#16) */
#user-card .bio, #group-card .bio {
    margin-top: unset !important;
}
```
makes the issue go away:

![image](https://user-images.githubusercontent.com/453543/51088144-9f62e780-175c-11e9-976a-c1731681fe47.png)

**Comment by amotl** [comment]
Hi @sesemaya,

thanks for getting back.
> I'm assuming this is happening in the popup user info dialog after you click a user's avatar

Correct.

As we are running some modifications on top of your theme, we just double-checked whether this will also happen with the plain vanilla version without any other theme components activated. It does:

![image](https://user-images.githubusercontent.com/453543/51290148-49639d80-1a03-11e9-8ce6-dbce4df5948d.png)

You can check this at the Discourse instance we are currently working on at https://meta.ip-tools.org/ to see it live. As far as we can recognize, it happens on every page.

It's not a serious thing though, as we could mitigate it with the simple fix on the CSS level as outlined above. So, it will be a matter of curiosity to dig deeper on this. Let us know if we can help.

Thanks,
Andreas.

P.S.: Please not that, while the site is currently on DMT vanilla w/o theme components, we will switch back to the [modified variant](https://github.com/ip-tools/discourse-material-theme-ample) soon.

**Comment by amotl** [comment]
Hi again,

we just found out it will only happen when Profile » About me is non-empty, as with the user profile of "discobot" and all others who configured it.

When this field is left empty, the rendering is also fine:
![image](https://user-images.githubusercontent.com/453543/51303213-f7d40680-1a34-11e9-84d9-7495e322ed27.png)

With kind regards.

**Comment by sesemaya** [comment]
Confirming this is indeed a bug. I'll get this fixed.

**Comment by sesemaya** [comment]
This is fixed by 04b7e8b.
